src/backend/gl/src/device.rs

```diff
@@ -290,9 +324,11 @@ impl Device {
         let version = self.share.info.shading_language.tuple();
         compile_options.version = if is_embedded {
             match version {
-                (3, 00) => glsl::Version::V3_00Es,
+                (3, 20) | (3, 2) => glsl::Version::V3_20Es,
```

Comment:
does this require a new spirv_cross?
@grovesNL could we get it on crates, please?

Reply:
My bad. Forgot about `cargo update`. 0.22.2 already released. Ready to merge.

Reply:
so now the backend requires `0.22.2`, we need to express this in the Cargo manifest accordingly

Reply:
You mean exactly `0.22.2` ?

We already set `0.22` in Cargo manifest.

```toml
spirv_cross = { version = "0.22", features = ["glsl"]}
```

Reply:
right, with your PR it should now be "0.22.2", so that people don't get the breakage when they update.